Dockerfile.jenkins

```diff
@@ -5,7 +5,7 @@ FROM golang:1.12.7
 RUN curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
 RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
 
-RUN apt-get update && apt-get install -y -qq postgresql-11 redis-server unzip
+RUN apt-get update && apt-get install -y -qq postgresql-11 redis-server unzip libuv1-dev libjson-c-dev nettle-dev
```

Comment:
Why is nettle still required as dependency?

Reply:
The cli uses it to encrypt the apikey and encryption key and save it to a file